### ISSUE TITLE
Latest AdMob including migration to SKAdNetwork [Crucial for all to continue displaying ads]

### ISSUE THREAD

**Issue by gigafishapps** [ratson/cordova-plugin-admob-free#473]
Google announced it will discontinue IDFA on all iOS devices using AdMob to take full advantage of Apple's SKAdNetwrok, so iOS apps won't need to display popups. My question is, will the latest version of the AdMob plugin be implemented so we can continue displaying any Ads on iOS?

**Comment by mattiaa95** [comment]
This may work, in the config.xml , inside `<platform name="ios">`


`<config-file target="*-Info.plist" parent="SKAdNetworkItems">`
 `            <array>`
  `                <dict>`
   `                <key>SKAdNetworkIdentifier</key>`
   `                <string>cstr6suwn9.skadnetwork</string>`
  `            </dict>`
 `        </array>`
`</config-file>`


and use 7.67 https://github.com/ratson/cordova-plugin-admob-free/issues/418#issuecomment-624599189

tell me if it works for you

**Comment by gigafishapps** [comment]
Thank you for your reply. It does not work for me at least. I strongly suggest producing a quick noob-friendly guide on the SKAdNetwork aspect as many people may suffer because of that not even knowing. 

The other solution is to drop this plugin and just generate an iOS app that serves as an HTML/Cordova wrapper and serve ads with the latest Google AdMob plugin.

But I guess this isn't a solution for all, as some people use Cordova because they do not use native iOS.

Nonetheless, this is a red flag issue for anyone thinking seriously about retaining their revenue stream on iOS with this plugin unless an in-house fix is produced.

**Comment by martijnmichel** [comment]
> This may work, in the config.xml , inside `<platform name="ios">`
> 
> `<config-file target="*-Info.plist" parent="SKAdNetworkItems">`
> ` <array>`
> ` <dict>`
> ` <key>SKAdNetworkIdentifier</key>`
> ` <string>cstr6suwn9.skadnetwork</string>`
> ` </dict>`
> ` </array>`
> `</config-file>`
> 
> and use 7.67 [#418 (comment)](https://github.com/ratson/cordova-plugin-admob-free/issues/418#issuecomment-624599189)
> 
> tell me if it works for you

It's been nearly a week since I've updated my app with the latest Admob SDK and I sill get the same message..

**Comment by mattiaa95** [comment]
> > This may work, in the config.xml , inside `<platform name="ios">`
> > `<config-file target="*-Info.plist" parent="SKAdNetworkItems">`
> > ` <array>`
> > ` <dict>`
> > ` <key>SKAdNetworkIdentifier</key>`
> > ` <string>cstr6suwn9.skadnetwork</string>`
> > ` </dict>`
> > ` </array>`
> > `</config-file>`
> > and use 7.67 [#418 (comment)](https://github.com/ratson/cordova-plugin-admob-free/issues/418#issuecomment-624599189)
> > tell me if it works for you
> 
> It's been nearly a week since I've updated my app with the latest Admob SDK and I sill get the same message..

have you checked if when compiling it is added in the info.plist, I still get it too :/ . let's see if someone can solve this problem.

**Comment by martijnmichel** [comment]
Does anyone know a solution?

**Comment by gigafishapps** [comment]
Yes, there is another plugin (new version of this one): https://admob-plus.github.io/docs/cordova/  You must remove the old plugin and install this one in your app. This should take care of the problem and you will be able to start using SKAdNetwork.

**Comment by martijnmichel** [comment]
@gigafishapps  That plugin seems to use AndroidX which gives me loads of other errors :S

**Comment by proofles** [comment]
any update?
